cmd/stateql/lib/lib.go

```diff
@@ -11,7 +11,7 @@ import (
 	"time"
 
 	"github.com/filecoin-project/statediff"
-	sdlib "github.com/filecoin-project/statediff/lib"
+	iface "github.com/filecoin-project/statediff/lib/interface"
 	"github.com/graphql-go/graphql"
 	"github.com/ipfs/go-cid"
 	blockstore "github.com/ipfs/go-ipfs-blockstore"
```

Comment:
Perhaps here and elsewhere go with the filecoin-core one ( unless this causes cycles of some sort...? )

```suggestion
	blockstore "github.com/filecoin-project/lotus/blockstore"
```

As [per docs](https://pkg.go.dev/github.com/filecoin-project/lotus/blockstore#pkg-overview):
> This package re-exports parts of the go-ipfs-blockstore package such that no other package needs to import it directly, for ergonomics and traceability.